openedx/features/cms/helpers.py

```diff
@@ -50,6 +53,8 @@ def set_rerun_course_module_dates(source_course, re_run, user):
     source_course_sections.extend(source_course_subsections)
     re_run_sections.extend(re_run_subsections)
 
+    set_ora_dates(re_run_subsections, source_course_start_date, re_run_start_date, user)
```

Comment:
if we're updating all dates from this method, then lets change the name of this method and lets also take the logic of setting course module dates in a new method